src/assets/stylesheets/extensions/_nav-multistep.scss

```diff
@@ -0,0 +1,154 @@
+@import "../config";
+
+.nav-multistep {
+  margin-top: 2rem;
+
+  ol {
+    display: flex;
+    margin-left: 0;
+    margin-bottom: 1rem;
+
+    li {
+      position:relative;
+      margin: 0;
+      list-style-type: none;
+      background: $codacy-neutral-800;
+      line-height: 1.25;
+
+      a,
+      p {
+        display: flex;
+        align-items: center;
+        width: 100%;
+        padding: 0 1rem;
+        color: #FFFFFF;
+      }
+
+      &.nav-multistep__step--active {
+        background: $codacy-neutral-300;
+
+        p {
+          font-weight: 700;
+          color: $codacy-neutral-800;
+        }
+      }
+
+      &:not(.nav-multistep__step--active):hover {
+        background: $codacy-neutral-700;
+      }
+    }
+  }
+}
+
+@include break-to-device(mobile landscape) {
+  .nav-multistep {
+    ol {
+      flex-direction: column;
+      li {
+        a,
+        p {
+          padding: 0.5rem 1rem;
+          margin: 0;
+        }
+
+        &:first-child,
+        &:nth-child(2) {
+          margin-bottom: 0.125rem;
+
+          &::after {
+            border-top: 0.375rem solid #FFFFFF;
+            border-right: 0.375rem solid transparent;
+            border-left: 0.375rem solid transparent;
+            position: absolute;
+            display: block;
+            content: "";
+            bottom: -0.5rem;
+            left: 1.5rem;
+            z-index: 1;
+          }
+        }
+      }
+    }
+  }
+}
+
+@include break-from-device(tablet portrait) {
+  .nav-multistep {
+    ol {
+      li {
+        display: flex;
+        flex: 0 0 33%;
+        height: 3rem;
+
+        &.nav-multistep__step--active {
+          &:first-child,
+          &:nth-child(2) {
+            &::after {
+              border-left-color: $codacy-neutral-300 !important;
+            }
+          }
+
+          &:nth-child(2),
+          &:nth-child(3) {
+            &::before {
+              border-top-color: $codacy-neutral-300 !important;
+              border-bottom-color: $codacy-neutral-300 !important;
+            }
+          }
```

Comment:
```suggestion
          &::before {
            border-top-color: $codacy-neutral-300 !important;
            border-bottom-color: $codacy-neutral-300 !important;
          }
          
          &::after {
            border-left-color: $codacy-neutral-300 !important;
          }
          
          &:last-child {
            &::after {
              border-left-color: transparent !important;
            }
          }

          &:first-child {
            &::before {
              border-top-color: transparent !important;
              border-bottom-color: transparent !important;
            }
          }
```

like this maybe?

Reply:
you can apply this logic in the other things bellow 👀 
The idea is to avoid using static indexes and limit you to use only 3 steps, this way in the future you can use as many steps as you want. You just need to design "middle steps" as default, and override the style of the both ends